src/Pages/Withdrawals.razor

```diff
@@ -160,38 +160,42 @@
                             }
                         </EditTemplate>
                     </DataGridColumn>
-                    <DataGridColumn TItem="WalletWithdrawalRequest" Filterable="false" PopupFieldColumnSize="ColumnSize.Is12" Editable="true" Field="@nameof(WalletWithdrawalRequest.Amount)" Caption="Amount (BTC)" Sortable="false" Displayable="@IsPendingRequestsColumnVisible(PendingWithdrawalsColumnName.Amount)" >
+                    <DataGridColumn TItem="WalletWithdrawalRequest" Filterable="false" PopupFieldColumnSize="ColumnSize.Is12" Editable="true" Field="@nameof(WalletWithdrawalRequest.Amount)" Caption="Amount" Sortable="false" Displayable="@IsPendingRequestsColumnVisible(PendingWithdrawalsColumnName.Amount)" >
                         <DisplayTemplate>
                             @{
                                 @($"{context.Amount:f8} BTC ({Math.Round(PriceConversionService.BtcToUsdConversion(context.Amount, _btcPrice), 2)} USD)")
                             }
                         </DisplayTemplate>
                         <EditTemplate>
-                            <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args)">
-                                <NumericPicker TValue="decimal" @bind-Value="@_amount" Step="0.00001m" CurrencySymbol="₿ " Max="@(_maxWithdrawal)" Min="_minWithdrawal" Decimals="8"  Disabled="@(SelectedUTXOs.Count > 0 || IsCheckedAllFunds)"  />
-                                     <FieldHelp>
+                            <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args, _selectedRequestWalletBalance)">
+                                <NumericPicker TValue="decimal" @bind-Value="@_amount" Step="0.00001m" CurrencySymbol="₿ " Decimals="8" DecimalSeparator="," Disabled="@(SelectedUTXOs.Count > 0 || IsCheckedAllFunds)">
+                                    <Feedback>
+                                        <ValidationError/>
```

Comment:
How does it look the ValidationError plus the FieldHelp? Because I remember weird things happening.

Reply:
It displaces the FieldHelp down and the error is displayed between the input and the FieldHelp. There is no straightforward way to put it below the FieldHelp because the Feedback tag has to be inside the input

Reply:
Okay, thats perfect. Just asking because I remembered someone saying something about it. 😄